queue_services/entity-emailer/requirements.txt

```diff
@@ -79,6 +79,15 @@ Werkzeug==1.0.1
 yarl==1.8.2
 zipp==3.15.0
 git+https://github.com/bcgov/business-schemas.git@2.18.27#egg=registry_schemas
-git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+
+# install updated legal-api from feature branch for development, 
+# TODO - change it back when the db versioning feature branch merged to main
+git+https://github.com/bcgov/lear.git@feature-db-versioning#egg=legal_api&subdirectory=legal-api
+# git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
```

Comment:
@leodube-aot how about the legal-api package, installing from the feature branch?

Reply:
Yup we'll be installing the legal-api package from the feature branch